src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExtensions.cs

```diff
@@ -25,35 +25,48 @@ namespace Dynatrace.OpenTelemetry.Exporter.Metrics
 	internal static class DynatraceMetricsExtensions
 	{
 		public static DynatraceMetric ToLongCounterDelta(this Metric metric, MetricPoint metricPoint, ILogger logger)
-			=> DynatraceMetricsFactory.CreateLongCounterDelta(
+		{
+			EnsureDeltaTemporality(metric);
```

Comment:
Hum.. do we need this? If we configure the reader with the preferred temporarily can we even get cumulative values here? Also throwing exceptions is probably not great as well, as we are only using it to sort of "control the flow".

If we want to check this, maybe returning `null` is better and checking it/logging it on the call site is better? If we log, maybe a throttled log of sorts would also be better to avoid bloating the logs of the app.

Reply:
Users can still create a `DynatraceExporter` and add it to a `MetricReader` themselves, so it is possible to misconfigure that one, so I think checking is still necessary.

Reply:
I see. Maybe then we should consider making the exporter private and only offer using it via the extension methods. I mean what is the point in offering it if users can misconfigure and produce data we don't like?